docsite/rst/galaxy.rst

```diff
@@ -169,7 +181,9 @@ The search command will return a list of the first 1000 results matching your se
 
 .. note::
 
-    The format of results pictured here is new in Ansible 2.0.
+    The search command in Ansible 2.0 requires using the Galaxy 2.0 Beta site. Use the ``--server`` option to access 
+    `https://galaxy-qa.ansible.com <https://galaxy-qa.ansible.com>`_. You can also add a *server* definition in the [galaxy] 
```

Comment:
It is available now using https://galaxy-qa.ansible.com. We are doing a beta release of the new Galaxy site to be announced officially between now and Monday morning. That's the point of the note.